PrePaper/2022/Curve块存储应用实践一部曲之iscsi(含开源tgt性能优化).md

```diff
@@ -0,0 +1,174 @@
+# 前言
+Curve 是云原生计算基金会 (CNCF) Sandbox 项目，同时提供分布式块存储以及分布式文件存储能力。
+
+为了让大家更容易使用以及了解Curve，我们期望接下来通过系列应用实践文章，以专题的形势向大家展示Curve。
+
+本篇文章是Curve块存储应用实践的第一篇，该系列文章包括:
+
+- Curve块存储应用实践一部曲之iscsi
+- Curve块存储应用实践二部曲之nbd
+- Curve块存储应用实践三部曲之云主机(OpenStack)
+- Curve块存储应用实践四部曲之云原生数据库
+- Curve块存储应用实践五部曲之性能调优
+
+
+
+# iscsi以及tgt的简介
+
+`tgt`是一个开源`iscsi`服务器，详情请见[tgt github](https://github.com/fujita/tgt)。我们在开发Curve块设备服务器时，想让更多的系统能够使用Curve块设备，而不仅仅是Linux系统，iscsi协议是一个广泛使用的块设备协议，我们想修改tgt以便让Curve提供iscsi服务。
+
+# curve块存储结合iscsi
+
+为tgt提供了访问curve的驱动，详见
+[doc/README.curve](https://github.com/opencurve/curve-tgt/tree/master/doc)
+， 文档里有操作步骤，这样用户就可以在任何支持iscsi的操作系统上使用curve块设备存储，例如Windows。
```

Comment:
https://github.com/opencurve/curveadm/wiki/curve-tgt-deployment#%E7%AC%AC-4-%E6%AD%A5%E5%90%AF%E5%8A%A8-tgtd-%E5%AE%88%E6%8A%A4%E8%BF%9B%E7%A8%8B

curveadm支持tgt的挂载方式，对比下这两个文档，看放哪个更合适

Reply:
或者部署考虑作为一小节，贴链接更突出？

Reply:
> https://github.com/opencurve/curveadm/wiki/curve-tgt-deployment#%E7%AC%AC-4-%E6%AD%A5%E5%90%AF%E5%8A%A8-tgtd-%E5%AE%88%E6%8A%A4%E8%BF%9B%E7%A8%8B
> 
> curveadm支持tgt的挂载方式，对比下这两个文档，看放哪个更合适

Yes, the documentation you provided is better. 

and I think links are more appropriate because they make the article more concise